src/AdoNetCore.AseClient/AseConnection.cs

```diff
@@ -536,6 +541,42 @@ private void NotifyTraceExit(object source, string method, object returnValue)
             get;
             set;
         }
+
+#if ENABLE_CLONEABLE_INTERFACE
+        public object Clone()
+        {
+            return new AseConnection(_connectionString, _connectionPoolManager);
+        }
+#endif
+
+        public void ClearPool()
+        {
+            _connectionPoolManager.ClearPool(_connectionString);
+        }
+
+        public bool IsCaseSensitive()
+        {
+            return _internal?.StatisticsEnabled ?? false;
```

Comment:
What does StatisticsEnabled have to do with case sensitivity?

Reply:
I think maybe that should be `_internal?.IsCaseSensitive ?? false;`. Woops!